release/restcomm-assembly-as7/pom.xml

```diff
@@ -0,0 +1,257 @@
+                            </images>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <id>start</id>
+                                <phase>pre-integration-test</phase>
+                                <goals>
+                                    <!-- "build" should be used to create the images with the
+                                         artifact -->
+                                    <goal>build</goal>
+                                    <!--<goal>start</goal> not running any containers yet-->
+                                </goals>
+                            </execution>
+                            <execution>
+                                <id>stop</id>
+                                <phase>post-integration-test</phase>
+                                <goals>
+                                    <!--<goal>stop</goal>   not running any containers yet -->
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>standalone</id>
+            <dependencies>
+                <dependency>
+                    <groupId>org.restcomm</groupId>
+                    <artifactId>restcomm-connect.application</artifactId>
+                    <type>war</type>
+                </dependency>
+
+                <dependency>
+                    <groupId>org.restcomm</groupId>
+                    <artifactId>restcomm-connect-management</artifactId>
+                    <type>war</type>
+                </dependency>
+            </dependencies>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-dependency-plugin</artifactId>
+                        <version>3.0.1</version>
+                        <executions>
+                            <execution>
+                                <id>extractSipServlets</id>
+                                <phase>prepare-package</phase>
+                                <goals>
+                                    <goal>unpack</goal>
+                                </goals>
+                                <configuration>
+                                    <artifactItems>
+                                        <artifactItem>
+                                            <groupId>org.mobicents.servlet.sip</groupId>
+                                            <artifactId>sip-servlets-as7</artifactId>
+                                            <version>${sipservletapi.version}</version>
+                                            <type>zip</type>
+                                            <classifier>assembly</classifier>
```

Comment:
classifier is the same as previous assembly, will this create collision if both profiles are enabled?

Reply:
i get your point, but - to make sure I did get it right - this is not the right place to add that comment, right? 

(i mean this right here is referring to the sip-servlets dependency, not the artifact generation... )

Reply:
right, this is not right place sorry, i guess you got the right point anyway..

Reply:
👍